linux/user-and-file-management/file-permissions/set-file-permissions-for-users.md

```diff
@@ -1,28 +1,13 @@
 ---
 author: nene
 
-levels:
-
-  - basic
-
-  - medium
-
-  - beginner
-
 aspects:
   - introduction
 
 type: normal
 
 category: must-know
 
-tags:
```

Comment:
why remove these, could be useful in the future?

Reply:
I figured YAGNI? They're not used in our system and most of the content has them with duplicate values of the aspects. If we ever add support for them, we'll need to re-tag the content anyhow 🤷‍♂